Tests/XMTPTests/ConversationsTest.swift

```diff
@@ -78,4 +78,64 @@ class ConversationsTests: XCTestCase {
 
 		await waitForExpectations(timeout: 3)
 	}
+    
+    func testCanValidateTopicsInsideConversation() async throws {
+        let validId = "sdfsadf095b97a9284dcd82b2274856ccac8a21de57bebe34e7f9eeb855fb21126d3b8f"
+        let fixtures = await fixtures()
+        
+        // Creation of all known types of topics
+        let privateStore = Topic.userPrivateStoreKeyBundle(validId).description
+        let contact = Topic.contact(validId).description
+        let userIntro = Topic.userIntro(validId).description
+        let userInvite = Topic.userInvite(validId).description
+        let directMessageV1 = Topic.directMessageV1(validId, "sd").description
+        let directMessageV2 = Topic.directMessageV2(validId).description
+        let preferenceList = Topic.preferenceList(validId).description
+        let conversations = fixtures.bobClient.conversations
```

Comment:
If we moved `isValidTopic` to `Topic` then we wouldn't need to bring in fixtures or deal with awaits below.